src/sys/statfs.rs

```diff
@@ -23,7 +23,7 @@ pub type fsid_t = libc::__fsid_t;
 pub type fsid_t = libc::fsid_t;
 
 cfg_if! {
-    if #[cfg(any(linux_android, target_os = "fuchsia"))] {
+    if #[cfg(any(linux_android, target_os = "fuchsia", target_env = "ohos"))] {
```

Comment:
If this `ohos` is a Linux libc, isn't this already included in `linux_android`?

Reply:
Nice catch!

It is a bit awkward that ohos is kind of target_env, but not target_os like android. Anyway, I fixed it along with another two places. 

Tested:
```
cargo build --target aarch64-unknown-linux-ohos --features acct,aio,dir,env,fs
```